pom.xml

```diff
@@ -84,6 +84,7 @@
 		<module>spring-cloud-dataflow-platform-kubernetes</module>
 		<module>spring-cloud-dataflow-platform-cloudfoundry</module>
 		<module>spring-cloud-starter-dataflow-server-local</module>
+		<module>spring-cloud-dataflow-audit</module>
```

Comment:
Forget to mention that new module should also go to `spring-cloud-dataflow-dependencies/pom.xml`

Reply:
In the new commit I added the audit dependenci into your mentioned `spring-cloud-dataflow-dependencies/pom.xml`